api/python/cellxgene_census/src/cellxgene_census/_open.py

```diff
@@ -76,6 +86,8 @@ def open_soma(
     Args:
```

Comment:
Other changes that need to be added:
* there are now more errors, e.g., mirror unknown, provider unsupported, etc.
* need example using mirror